docs/signed-apk-android.md

```diff
@@ -93,7 +93,7 @@ _Note: In order for Google Play to accept AAB format the App Signing by Google P
 
 ### Testing the release build of your app
 
-Before uploading the release build to the Play Store, make sure you test it thoroughly. First uninstall any previous version of the app you already have installed. Install it on the device using:
+Before uploading the release build to the Play Store, make sure you test it thoroughly. First uninstall any previous version of the app you already have installed. Install it on the device using(in project root):
```

Comment:
Space is missing between last word and round bracket.

What do you think about putting this information into the sentence? 
* "Install it on the device using the following command in the project root:"

Reply:
Any language advice is accepted, as I am not an English speaker

Reply:
I'm not the one too, thank you for updating this PR during review!